protos/flyteidl/event/event.proto

```diff
@@ -154,6 +154,14 @@ message ParentNodeExecutionMetadata {
     string node_id = 1;
 }
 
+message Reason {
+    // An explanation for this event
+    string reason = 1;
```

Comment:
sorry to be so nitpicky - but won't this cause a stutter when accessing, e.g. `Reason.GetReason()` ?

either way don't feel too strongly

Reply:
I've been waffling on this too. I didn't want to call this an `Event` because it's in an event. Maybe `EventReason.GetReason()` or `ReasonInfo.GetReason()`. Have a preference?

Reply:
both sound great! maybe slight preference for the former